src/Arcus.Scripting.IntegrationAccount/Scripts/Set-AzIntegrationAccountCertificates.ps1

```diff
@@ -0,0 +1,95 @@
+    )
+
+    $certificateName = $Certificate.BaseName
+    if ($ArtifactsPrefix -ne '') {
+        $certificateName = $ArtifactsPrefix + $certificateName
+    }
+    Write-Host "Uploading certificate '$certificateName' into the Integration Account '$Name'"
+
+    $existingCertificate = $null
+    try {
+        Write-Verbose "Checking if the certificate '$certificateName' already exists in the Integration Account '$Name'"
+        $existingCertificate = Get-AzIntegrationAccountCertificate -ResourceGroupName $ResourceGroupName -IntegrationAccount $Name -CertificateName $certificateName -ErrorAction Stop
+    }
+    catch {
+        if ($_.Exception.Message.Contains('could not be found')) {
+            Write-Verbose "No certificate '$certificateName' could not be found in Azure Integration Account '$Name'"
+        }
+        else {
+            throw $_.Exception
+        }
+    }
+        
+    try {
+        if ($existingCertificate -eq $null) {
+            Write-Verbose "Creating certificate '$certificateName' in Azure Integration Account '$Name'"
+            if ($CertificateType -eq 'Public') {
+                $createdCertificate = New-AzIntegrationAccountCertificate -ResourceGroupName $ResourceGroupName -IntegrationAccount $Name -CertificateName $certificateName -PublicCertificateFilePath $Certificate.FullName -ErrorAction Stop
+            }
+            else {
+                $createdCertificate = New-AzIntegrationAccountCertificate -ResourceGroupName $ResourceGroupName -IntegrationAccount $Name -CertificateName $certificateName -PublicCertificateFilePath $Certificate.FullName -KeyName $KeyName -KeyVersion $KeyVersion -KeyVaultId $KeyVaultId -ErrorAction Stop
+            }
+            Write-Verbose ($createdCertificate | Format-List -Force | Out-String)
+        }
+        else {
+            Write-Verbose "Updating certificate '$certificateName' in Azure Integration Account '$Name'"
+            if ($CertificateType -eq 'Public') {
+                $updatedCertificate = Set-AzIntegrationAccountCertificate -ResourceGroupName $ResourceGroupName -IntegrationAccount $Name -CertificateName $certificateName -PublicCertificateFilePath $Certificate.FullName -Force -ErrorAction Stop
+            }
+            else {
+                $updatedCertificate = Set-AzIntegrationAccountCertificate -ResourceGroupName $ResourceGroupName -IntegrationAccount $Name -CertificateName $certificateName -PublicCertificateFilePath $Certificate.FullName -KeyName $KeyName -KeyVersion $KeyVersion -KeyVaultId $KeyVaultId -Force -ErrorAction Stop
+            }
+            Write-Verbose ($updatedCertificate | Format-List -Force | Out-String)
+        }
+        Write-Host "Certificate '$certificateName' has been uploaded into the Azure Integration Account '$Name'"
+    }
+    catch {
+        Write-Error "Failed to upload certificate '$certificateName' in Azure Integration Account '$Name': '$($_.Exception.Message)_'"
```

Comment:
There's a trailing underscore here? Not sure why? 😄

Reply:
Because why not 😉 
Fixed in all other scripts as well (schemas, maps, assemblies), see [`9dcf966` (#259)](https://github.com/arcus-azure/arcus.scripting/pull/259/commits/9dcf966dfa6e82e333377cf843ffa4b4f6f7301c)